WARNING 2025-03-07 01:16:56,804 log 97732 6197325824 Not Found: /apps/invoice
WARNING 2025-03-07 01:18:00,449 log 97732 6214152192 Not Found: /apps/invoice
WARNING 2025-03-07 01:18:17,798 log 97732 6197325824 Not Found: /apps/invoice
WARNING 2025-03-07 01:20:13,670 log 97732 6247804928 Not Found: /apps/invoice
WARNING 2025-03-07 01:20:23,351 log 97732 6197325824 Not Found: /apps/invoice
WARNING 2025-03-07 01:22:18,011 log 97732 6230978560 Not Found: /apps/invoice
WARNING 2025-03-07 01:33:29,569 log 97732 6281457664 Not Found: /apps/invoice
WARNING 2025-03-07 01:44:59,132 log 97732 6197325824 Not Found: /apps/invoice
WARNING 2025-03-07 05:31:19,310 log 97732 6197325824 Not Found: /apps/users/undefined
WARNING 2025-03-07 05:55:31,313 log 97732 6197325824 Not Found: /apps/users
INFO 2025-03-07 14:37:29,383 autoreload 97732 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/use_cases/role_use_case.py changed, reloading.
INFO 2025-03-07 14:37:29,825 autoreload 67485 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:49:29,325 autoreload 67485 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:49:29,686 autoreload 80154 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:52:29,138 autoreload 80154 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:52:29,462 autoreload 83315 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:54:45,073 autoreload 83315 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:54:45,492 autoreload 85696 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:55:34,003 autoreload 85696 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:55:34,341 autoreload 86548 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:56:04,284 autoreload 86548 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/use_cases/user_use_case.py changed, reloading.
INFO 2025-03-07 14:56:04,645 autoreload 87072 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:57:45,452 autoreload 87072 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:57:45,797 autoreload 88868 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:57:52,153 autoreload 88868 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:57:52,438 autoreload 88991 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:58:30,571 autoreload 88991 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:58:30,953 autoreload 89671 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:58:35,248 autoreload 89671 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:58:35,437 autoreload 89788 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:59:13,459 autoreload 89788 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/user_serializer.py changed, reloading.
INFO 2025-03-07 14:59:13,753 autoreload 90452 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 14:59:54,979 autoreload 90452 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 14:59:55,289 autoreload 91204 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 15:42:20,779 autoreload 91204 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 15:42:21,169 autoreload 36033 8257288256 Watching for file changes with StatReloader
ERROR 2025-03-07 15:42:36,672 exception 36033 6169849856 Ocurrió una excepción: 'lastname'
ERROR 2025-03-07 15:42:36,674 exception 36033 6169849856 Detalles del traceback:
Traceback (most recent call last):
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/django/views/decorators/csrf.py", line 65, in _view_wrapper
    return view_func(request, *args, **kwargs)
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/django/views/generic/base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
           ~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "/Users/aemo/Projects/api_sataiga/api_sataiga/api/views/role.py", line 15, in get
    return use_case.get()
           ~~~~~~~~~~~~^^
  File "/Users/aemo/Projects/api_sataiga/api_sataiga/api/use_cases/role_use_case.py", line 51, in get
    RoleSerializer(page.object_list, many=True).data
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/serializers.py", line 795, in data
    ret = super().data
          ^^^^^^^^^^^^
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/serializers.py", line 249, in data
    self._data = self.to_representation(self.instance)
                 ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/serializers.py", line 714, in to_representation
    self.child.to_representation(item) for item in iterable
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/serializers.py", line 538, in to_representation
    ret[field.field_name] = field.to_representation(attribute)
                            ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^
  File "/Users/aemo/Projects/api_sataiga/venv/lib/python3.13/site-packages/rest_framework/fields.py", line 1870, in to_representation
    return method(value)
  File "/Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py", line 14, in get_users
    return [{'_id': str(u['_id']), 'name': u['name'], 'lastname': u['lastname'], 'email': u['email']} for u in users]
                                                                  ~^^^^^^^^^^^^
KeyError: 'lastname'

ERROR 2025-03-07 15:42:36,674 log 36033 6169849856 Internal Server Error: /api/roles
INFO 2025-03-07 15:43:30,134 autoreload 36033 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/serializers/role_serializer.py changed, reloading.
INFO 2025-03-07 15:43:30,484 autoreload 37259 8257288256 Watching for file changes with StatReloader
INFO 2025-03-07 16:06:40,296 autoreload 37259 8257288256 /Users/aemo/Projects/api_sataiga/api_sataiga/api/models/role.py changed, reloading.
INFO 2025-03-07 16:06:40,662 autoreload 61848 8257288256 Watching for file changes with StatReloader
